code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java

```diff
@@ -14,26 +14,24 @@
 import static com.adobe.marketing.mobile.edge.bridge.EdgeBridgeConstants.LOG_TAG;
 
 import com.adobe.marketing.mobile.Event;
+import com.adobe.marketing.mobile.EventSource;
+import com.adobe.marketing.mobile.EventType;
 import com.adobe.marketing.mobile.Extension;
 import com.adobe.marketing.mobile.ExtensionApi;
-import com.adobe.marketing.mobile.ExtensionError;
-import com.adobe.marketing.mobile.ExtensionErrorCallback;
-import com.adobe.marketing.mobile.LoggingMode;
-import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.services.Log;
+import com.adobe.marketing.mobile.util.DataReader;
+import com.adobe.marketing.mobile.util.StringUtils;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Locale;
 import java.util.Map;
 import java.util.TimeZone;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
 
 class EdgeBridgeExtension extends Extension {
 
-	private final String SELF_LOG_TAG = "EdgeBridgeExtension";
-	private ExecutorService executorService;
-	private final Object executorMutex = new Object();
+	private static final String LOG_SOURCE = "EdgeBridgeExtension";
+	public static final Class<? extends Extension> EXTENSION = EdgeBridgeExtension.class;
```

Comment:
nit: suggest re-ordering `public static final` members ahead of `private static final`